cterasdk/core/users.py

```diff
@@ -107,21 +107,22 @@ def add(self, name, email, first_name, last_name, password, role, company=None,
 
         return response
 
-    def modify(self, name, email=None, first_name=None,
+    def modify(self, username, name=None, email=None, first_name=None,
```

Comment:
I had to think what's the name parameter for. Since there are so many parameters with "name" in it, maybe the first one should be current_username and the second one should be new_username. This way the parameter names are more intuitive.
What do you think?